pages/cpm.vue

```diff
@@ -1,9 +1,15 @@
 <template>
   <div>
-    <h1 class="text-2xl m-4">
-      Disclaimer: This page is a work in progress and does not currently contain dependable climate information!
-    </h1>
-    <div class="flex gap-4 m-4">
+    <!-- breadcrumbs -->
+    <div class="m-4 ml-8 text-lg">
+      <Breadcrumbs :crumbs="crumbs" />
+    </div>
+    <!-- brief summary -->
+    <p class="m-2 ml-8 text-lg">
+      Select a region on the map to check the CPM model & data availability and explore the datasets in the panels.
```

Comment:
Building upon [CORDEX FPS-Convection](), EUCP has performed convection-permitting model runs for multiple domains covering Europe and the outermost regions. This page provides an overview of the most relevant data and results from our preliminary analysis. Select a region on the map to start exploring.

Reply:
Thanks for the text! I will keep it for now and adapt it later to the new design.